test/functional/test_short_deck.py

```diff
@@ -170,3 +171,52 @@ def test_pre_flop_pot(n_players: int, small_blind: int, big_blind: int):
     assert (
         n_bet_chips == pot.total
     ), f"small and big blind have are not in pot! {n_bet_chips} == {pot.total}"
+
+
+def _play_game_helper(state, betting_round_dict, bad_seq):
+    p = 1 / len(state.legal_actions)
+    probabilities = np.full(len(state.legal_actions), p)
+    a = np.random.choice(state.legal_actions, p=probabilities)
+    betting_stage = state._betting_stage
+    if betting_stage not in ["show_down", "terminal"]:
```

Comment:
In future (not a big deal here as just 2 els), prefer a set `{"show_down", "terminal"}` due to O(1) lookup time

Reply:
Good call, thanks for that